.github/actions/build-refs/dist/index.js

```diff
@@ -0,0 +1,9898 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
do we want to keep this generated code?

Reply:
Yes. This is unfortunately how this process works - github won't autogenerate anything for you, you have to check the javascript file you want to run into the repo. Since we want to have the actual code be in typescript and transpile to javascript, we have to do that and check the code in. It's marked as a binary file which apparently github doesn't respect, which is annoying - your local git client won't show a diff here.